server/app/views/admin/TranslationFormView.java

```diff
@@ -93,4 +100,28 @@ protected <T extends Tag<T>> FormTag renderTranslationForm(
                     .withId("update-localizations-button"));
     return form;
   }
+
+  /**
+   * Returns a div containing the default text to be translated. This allows for admins to more
+   * easily identify which text to translate.
+   */
+  protected final DivTag defaultLocaleTextHint(LocalizedStrings localizedStrings) {
+    return div()
+        .withClasses(Styles.W_2_3, Styles.ML_2, Styles.P_2, Styles.TEXT_SM, Styles.BG_GRAY_200)
+        .with(p("Default text:").withClass(Styles.FONT_MEDIUM), p(localizedStrings.getDefault()));
+  }
+
+  /** Creates a fieldset wrapping several form fields to be rendered. */
+  protected final FieldsetTag fieldSetForFields(
+      String legendText, ImmutableList<DomContent> fields) {
+    return fieldset()
+        .withClasses(Styles.MY_4, Styles.PT_1, Styles.PB_2, Styles.PX_2, Styles.BORDER)
+        .with(
+            legend(legendText), div().withClasses(Styles.FLEX_ROW, Styles.SPACE_Y_4).with(fields));
+  }
+
+  protected final boolean isDefaultLocale(Locale locale) {
+    // TODO(#2752): Remove isDefaultLocale once English translations have been disabled.
```

Comment:
This should be on/above the method not in it.

Reply:
Done!